dev-python/cloudscraper/cloudscraper-1.2.65.ebuild

```diff
@@ -31,6 +31,10 @@ BDEPEND="
 	)
 "
 
+# Most tests fail for no understandable reason
+# and it seems impossible to make the test suite skip them
```

Comment:
`EPYTEST_DESELECT`?

Reply:
You're right, this did work. Previously I tried adding annotation or calling pytest functions but these didn't work.